src/plugins/ImportArtifact/metadata.json

```diff
@@ -4,28 +4,29 @@
   "version": "0.1.0",
   "description": "",
   "icon": {
-    "class": "glyphicon glyphicon-cloud-upload",
+    "class": "glyphicon glyphicon-transfer",
     "src": ""
   },
   "disableServerSideExecution": false,
   "disableBrowserSideExecution": false,
-  "writeAccessRequired": true,
+  "writeAccessRequired": false,
   "configStructure": [
-        {
-            "name": "dataHash",
-            "displayName": "Data to upload",
-            "description": "",
-            "value": "",
-            "valueType": "asset",
-            "readOnly": false
-        },
-        {
-            "name": "dataTypeId",
-            "displayName": "Data Type",
-            "description": "",
-            "value": "",
-            "valueType": "str",
-            "readOnly": false
-        }
+    {
+      "name": "dataPath",
+      "displayName": "Path",
+      "description": "Path to an existing artifact in the storage backend",
+      "value": "",
+      "valueType": "str",
```

Comment:
It looks like this was also an error in the previous metadata but this should be `"string"` not `"str"`.

Reply:
Changed `ValueType` to string.